src/libs/SearchQueryUtils.ts

```diff
@@ -544,7 +544,8 @@ function getDisplayValue(filterName: string, filter: string, personalDetails: On
         return cardList[filter]?.bank || filter;
     }
     if (filterName === CONST.SEARCH.SYNTAX_FILTER_KEYS.IN) {
-        return ReportUtils.getReportName(reports?.[`${ONYXKEYS.COLLECTION.REPORT}${filter}`]) || filter;
+        const reportName = ReportUtils.getReportName(reports?.[`${ONYXKEYS.COLLECTION.REPORT}${filter}`]);
```

Comment:
Is this change relevant?

Reply:
its not, those were just my tests, thanks!